packages/OpenSqueakMap.package/OSMRenderConfigurationBrowser.class/instance/color.st

```diff
@@ -0,0 +1,5 @@
+actions
+color
+
+	self selectedAttribute: #color;
```

Comment:
minor thing: this way of remembering the selected attribute leads to not being able to edit two different color attributes at once because both of the color pickers will change the same value then. Not sure if this is desired behavior but nevertheless not a big issue...